alerts.php

```diff
@@ -28,32 +28,30 @@
  */
 
 use LibreNMS\Alert\RunAlerts;
-use LibreNMS\Util\FileLock;
 
 $init_modules = ['alerts', 'laravel'];
 require __DIR__ . '/includes/init.php';
 
 $options = getopt('d::');
 
-$alerts_lock = FileLock::lockOrDie('alerts');
-
-$alerts = new RunAlerts();
-
 if (set_debug(isset($options['d']))) {
     echo "DEBUG!\n";
 }
 
-if (! defined('TEST') && \LibreNMS\Config::get('alert.disable') != 'true') {
-    echo 'Start: ' . date('r') . "\r\n";
-    echo 'ClearStaleAlerts():' . PHP_EOL;
-    $alerts->clearStaleAlerts();
-    echo "RunFollowUp():\r\n";
-    $alerts->runFollowUp();
-    echo "RunAlerts():\r\n";
-    $alerts->runAlerts();
-    echo "RunAcks():\r\n";
-    $alerts->runAcks();
-    echo 'End  : ' . date('r') . "\r\n";
+$alerts_lock = Cache::lock('alerts');
```

Comment:
Missing use Cache?

Reply:
hehe, the ide doesn't like the use statement...  leaving as-is.